### DISCUSSION TITLE
docs generator

### DISCUSSION THREAD

**Discussion by ollyde** [trpc/trpc#2567] [Ideas]
### Describe the feature you'd like to request

Would be the perfect library if we had a documentation generator and some basic page explaining the functions, while using the express plugin for example we could have endpoints with their input/output models.

JSDocs (https://jsdoc.app/about-getting-started.html) is a good example of a simple doc generator. I have created TSDocs in the past (private project) which looks at models and automatically attaches them to the docs page, would be awesome if we have some similar functionality on tRPC.

### Describe the solution you'd like to see

^^

### Desribe alternate solutions

^^

### 👨‍👧‍👦 Contributing

- [ ] 🙋‍♂️ Yes, I'd be down to file a PR implementing this feature!

**Comment by mmkal** [comment]
https://api-extractor.com might be worth looking at. I haven't used it beyond light poking around but it does claim to do this sort of thing.

**Reply by ollyde** [reply]
@mmkal my solution is to use trpc playground. https://github.com/sachinraja/trpc-playground
